docs/visualization/plotting-overview.ipynb

```diff
@@ -232,38 +274,39 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "d['data_2d'].attrs['T'] = sc.Variable(dims=['x'], values=np.linspace(3, 50, M), unit=sc.units.K )\n",
-    "sc.plot(d['data_2d'], labels={'x': 'T'})"
+    "da1.plot(scale={'x': 'log'})"
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": null,
    "metadata": {},
+   "outputs": [],
    "source": [
-    "To control which dimensions are shown along which axes of the matplotlib figure use the transpose button in the toolbar, or transpose data before calling `plot`, e.g."
+    "da.plot(scale={'x': 'log', 'time': 'log'})"
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": null,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [],
    "source": [
-    "d['data_2d'].transpose().plot()"
+    "Note the the keys in the `scale` dict are dimension labels and *not* \"x\" and \"y\" as Matplotlib would refer to its axes."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "<div class=\"alert alert-info\">\n",
-    "\n",
-    "*Note*\n",
-    "\n",
-    "Plot settings are cached in a Scipp `config.yaml` file.\n",
-    "See <a href=\"../python-reference/runtime-configuration.ipynb\">runtime-configuration</a> for more information\n",
-    "\n",
-    "</div>"
+    "For the dependent axis (the data axis, i.e., vetical axis) use the `norm` option:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "(100 * (da1 - da1.min())).plot(norm='log')"
```

Comment:
Why the factor of 100?

Reply:
Because of your comment to have a nicer range on log scale.